gemfiles/Gemfile.rails-5.0-master

```diff
@@ -0,0 +1,33 @@
+source "https://rubygems.org"
+
+gemspec path: '..'
+
+gem "webmock", "~> 1.20.4"
+gem "bundler", "~> 1.6"
+gem "rake"
+gem "jquery-rails"
+gem "capybara", github: 'jnicklas/capybara'
+gem "pry"
+gem "pry-byebug", platforms: [:mri]
+gem "aws-sdk"
+gem "rack-test", "~> 0.6.2"
+gem "sqlite3",                          platforms: [:ruby]
+gem "activerecord-jdbcsqlite3-adapter", platforms: [:jruby]
+gem "poltergeist"
+gem "yard"
+gem "rubocop", "~>0.33.0"
+gem "puma"
+gem "mini_magick"
+gem "simple_form"
+gem "rails", github: "rails/rails"
+gem "arel", github: "rails/arel"
+gem "rack", github: "rack/rack"
+gem "rspec-rails", github: "rspec/rspec-rails"
+gem "rspec", github: "rspec/rspec"
+gem "rspec-core", github: "rspec/rspec-core"
+gem "rspec-support", github: "rspec/rspec-support"
+gem "rspec-expectations", github: "rspec/rspec-expectations"
+gem "rspec-mocks", github: "rspec/rspec-mocks"
+# swap back to the sinatra repo once the sinatra send_file issue is fixed
+# gem "sinatra", github: "sinatra/sinatra", branch: "2.2.0-alpha"
+gem "sinatra", github: "kgrz/sinatra", branch: "use-rack-files-serving-method"
```

Comment:
@twalpole; appears that https://github.com/sinatra/sinatra/pull/1045 has now been merged into [sinatra:2.2.0-alpha](https://github.com/sinatra/sinatra/tree/2.2.0-alpha).